config/dev-migrations/20211209101400-mock-change-events.graph

```diff
@@ -0,0 +1 @@
+http://mu.semte.ch/graphs/contacthub/141d9d6b-54af-4d17-b313-8d1c30bc3f5b/ChAdmin
```

Comment:
Not sure if this is still the correct graph after the role changes

Reply:
Indeed, should be http://mu.semte.ch/graphs/organisatieportaal